check_for_data.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Why do we need this file at all? We are going to user supervisorctl to invoke a management command. venv can be activated by invoking a venv python interpreter directly, like /home/deploy/binance-alarm/venv/bin/python.